cdm/src/test/java/ucar/nc2/ft2/coverage/simpgeometry/TestReadPoint.java

```diff
@@ -0,0 +1,47 @@
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import ucar.ma2.Array;
+import ucar.nc2.dataset.NetcdfDataset;
+import ucar.nc2.ft2.coverage.simpgeometry.*;
+import ucar.unidata.util.test.TestDir;
+
+import java.util.Random;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Point construction tests at the dataset level.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class TestReadPoint {
+
+	private SimpleGeometryReader newReader()
+	{
+		String filepath = TestDir.cdmLocalTestDataDir + "dataset/SimpleGeos/avg_temp_3gage_5timesteps.nc";
+		NetcdfDataset dataset = null;
+		
+		try {
+		
+			dataset = NetcdfDataset.openDataset(filepath);

```

Comment:
Would you be able to add an assertion to this test? Maybe just a not null assertion to start out?

Reply:
Oh yeah this test is not finished, still need working Point data and Line data. Will get both the Line and Point tests finished once those are fixed.

Reply:
I will add a Not Null Check though on the Reader, thanks!